cmd/podmanV2/containers/exec.go

```diff
@@ -0,0 +1,95 @@
+package containers
+
+import (
+	"bufio"
+	"os"
+
+	"github.com/containers/libpod/cmd/podmanV2/common"
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	envLib "github.com/containers/libpod/pkg/env"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	execDescription = `Execute the specified command inside a running container.
+`
+	execCommand = &cobra.Command{
+		Use:               "exec [flags] CONTAINER [COMMAND [ARG...]]",
+		Short:             "Run a process in a running container",
+		Long:              execDescription,
+		PersistentPreRunE: preRunE,
```

Comment:
```suggestion
		PreRunE: preRunE,
```
exec has not sub commands so you don't need Persistent